src/rockstor/storageadmin/static/storageadmin/js/views/setrole_disks.js

```diff
@@ -233,15 +233,15 @@ SetroleDiskView = RockstorLayoutView.extend({
             // Also serves to indicate no redirect role in operation.
             var uuid_message;
             if ( (this.disk_btrfs_uuid != null) && (_.isEmpty(this.partitions)) ) {
-                uuid_message = 'btrfs';
+                uuid_message = 'btrfs'
             } else {
                 uuid_message = 'None';
             }
             var selected = '';
             if (this.current_redirect == '') {
                 selected = ' selected="selected"';
             }
-            html += '<option value=""' + selected + '> Whole Disk (' + uuid_message + ')';
+            html += '<option value="' + selected + '"> Whole Disk (' + uuid_message + ')';
```

Comment:
@MFlyer I think this was 'as intended' ie not a typo as for a "Whole Disk" entry the value to be returned will = "" ie empty string. It's used as an indicator in the backend to signal no redirect: ie Whole Disk. Your change will break that mechanism. Can you double check your changes just in case. Ie first entry in this drop down list is 'hard wired' to have a value of "" to always offer the option to remove / (have no) redirect role defined.

It would be a shame to lose the option to define whole disk as an option :)